Carrot-Assistant/components/embeddings.py

```diff
@@ -30,6 +42,22 @@ def __init__(
             model: EmbeddingModel,
             search_kwargs: dict,
             ) -> None:
+        """
+        Initialises the connection to an embeddings database
+
+        Parameters
+        ----------
+        embeddings_path: str
+            A path for the embeddings database. If one is not found, it will be built, which takes a long time. This is built from concepts fetched from the OMOP database
```

Comment:
shall we point here to where one can find the OMOP database? I wonder if it would be good to explain what is meant by path here. Path to me is just a pointer to an address where the file is located, right? So why would this take long to run? perhaps it is not the path that will be built, but the database itself? Just maybe clarify?